packages/zowe-explorer/README.md

```diff
@@ -23,26 +23,27 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.11.0](#whats-new-in-zowe-explorer-1.11.0)
+- [What's new in Zowe Explorer 1.12.0](#whats-new-in-zowe-explorer-1.12.0)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
 - [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.11.0
+## What's new in Zowe Explorer 1.12.0
 
 New features:
 
-- Added login and logout functions for base profiles. You can now log in to API Mediation Layer and generate a token for your base profile.
+- Added the ability to edit data set attributes before allocation
+- Allowed filtering of member names from the Data Sets search bar
```

Comment:
Could we change it to "Allowed member filtering from the Data Sets search bar"?

It sounds a bit like it will limit what member names you can use or sth.

Reply:
The current version sounds like it will filter the names of a memeber of a chosen data set, which is a bit weird. Shouldn't it be somethig like "Added the member-filtering functionality to the Data Sets search bar"?